### PR TITLE
feat(service): init project and add models、entitities

### PR DESCRIPTION
**Pull request by codding-y**

### CODE REVIEW

File: src/Services/MASA.PM.Service.Admin/MASA.PM.Service.Admin.csproj [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	
+	<ItemGroup>
+		<PackageReference Include="MASA.BuildingBlocks.Dispatcher.Events" Version="0.0.1681936502" />
+		<PackageReference Include="MASA.Contrib.Data.Contracts.EF" Version="0.0.1706108591" />
+		<PackageReference Include="MASA.Contrib.Dispatcher.Events" Version="0.0.1685760477" />
+		<PackageReference Include="MASA.Contrib.Dispatcher.IntegrationEvents.EventLogs.EF" Version="0.0.1685760477" />
+		<PackageReference Include="Dapr.AspNetCore" Version="1.5.0" />
+		<PackageReference Include="MASA.Contrib.ReadWriteSpliting.CQRS" Version="0.0.1685760477" />
+		<PackageReference Include="MASA.Contrib.Service.MinimalAPIs" Version="0.0.1685760477" />
+		<PackageReference Include="MASA.Utils.Configuration.Json" Version="0.0.1711196569" />
```
Comment (doddgu) [inline src/Services/MASA.PM.Service.Admin/MASA.PM.Service.Admin.csproj:30]: 0.0.1711196569?

File: src/Services/MASA.PM.Service.Admin/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "profiles": {
+    "MASA.PM.Services": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:6030",
```
Comment (doddgu) [inline src/Services/MASA.PM.Service.Admin/Properties/launchSettings.json:18]: Why is port 6030?

**Comment by doddgu** [comment]
@capdiem Review the blazor.

**Review by capdiem — COMMENTED** [review]
lots of images in wwwroot from masa.blazor.pro template are not needed.

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**